### PR TITLE
Update billing docs for AM3

### PR DESCRIPTION
**Pull request by lizokm**
## DESCRIBE YOUR PR
Adding new information to billing docs.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ X ] Urgent deadline 
- [ ] Other deadline: 
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/pricing/index.mdx [hunk 1]
```diff
@@ -72,18 +72,17 @@ Performance Units can be used towards multiple Performance event types, dependin
...
-| >4.5M-10M | $0.001962 | $0.002551 |
+| >4.5M-20M | $0.001962 | $0.002551 |
 
 #### Cron Monitors Pricing
 
-Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a predetermined on-demand budget that can be shared across all event types or defined per category.
+Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a pay-as-you-go budget that can be shared across all event types.
 
 | Total Monitors Volume | Team/Business On-Demand |
 |---|---|
```
Comment (lizokm) [inline docs/pricing/index.mdx:88]:
```suggestion
 | Team Pay-as-you-go | Business Pay-as-you-go |
| $0.7800000 | $0.7800000 | 

#### Attachments Pricing

 | Total Attachments Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid | Business Pay-as-you-go |
| >1G | $0.2500000 | $0.3125000 | $0.2500000 | $0.3125000 | 

```

File: docs/pricing/index.mdx [hunk 2]
```diff
@@ -37,29 +42,24 @@ Please note that all per-category prices listed below have been rounded to the n
...
-| >25M-50M | $0.000120 | $0.000156 | $0.000144 | $0.000187 |
-
-#### Performance Pricing
+| >50k-100k | $0.0002900 | $0.0003625 | $0.0008900 | $0.0011125 |
+| >100k-500k | $0.0001750 | $0.0002188 | $0.0005000 | $0.0006250 |
+| >500k-2M | $0.0001500 | $0.0001875 | $0.0003000 | $0.0003750 |
+| >2M-10M | $0.0001500 | $0.0001875 | $0.0003000 | $0.0003750 |
+| >10M-20M | $0.0001300 | $0.0001625 | $0.0002600 | $0.0003250 |
+| >20M-25M | $0.0001200 | $0.0001500 | $0.0002400 | $0.0003000 |
+| >25M-50M | $0.0001200 | $0.000100 | $0.0002400 | $0.0003000 |
```
Comment (lizokm) [inline docs/pricing/index.mdx:53]:
```suggestion
| >25M | $0.0001200 | $0.000100 | $0.0002400 | $0.0003000 |
```

File: docs/pricing/index.mdx [hunk 3]
```diff
@@ -37,29 +42,24 @@ Please note that all per-category prices listed below have been rounded to the n
...
-| Total Performance Unit Volume | Team Prepaid | Team On-Demand | Business Prepaid | Business On-Demand |
+| Total Span Unit Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid | Business Pay-as-you-go |
 |---|---|---|---|---|
-| >100k-250k | $0.000100 | $0.000189 | $0.000300 | $0.000579 |
-| >250k-500k | $0.000058 | $0.000130 | $0.000178 | $0.000390 |
+| >10M-100M | $0.0000016 | $0.0000020 | $0.0000032 | $0.0000040 |
+| >100M-10B | $0.0000014 | $0.0000018 | $0.0000029 | $0.0000036 |
 | >500k-1M | $0.000052 | $0.000075 | $0.000130  | $0.000231 |
 | >1M-4.5M | $0.000052 | $0.000068 | $0.000130 | $0.000169 |
 | >4.5M-5M | $0.000046 | $0.000068 | $0.000092 | $0.0000169 |
```
Comment (lizokm) [inline docs/pricing/index.mdx:65]:
```suggestion
| >10M-100M | $0.0000016 | $0.0000020 | $0.0000032 | $0.0000040 |
| >100M | $0.0000014 | $0.0000018 | $0.0000029 | $0.0000036 |

```

File: docs/pricing/index.mdx [hunk 4]
```diff
@@ -72,21 +72,26 @@ Performance Units can be used towards multiple Performance event types, dependin
...
-| >100k-900k | $0.002565 | $0.003335 |
-| >900k-4.5M | $0.002309 | $0.003001 |
-| >4.5M-10M | $0.001962 | $0.002551 |
+| Total Replays Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid
+| Business Pay-as-you-go |
+| >5k-10k | $0.0028500 | $0.0035630 | $0.0028500 | $0.0035630 | 
+| >10k-100k | $0.0028500 | $0.0035630 | $0.0028500 | $0.0035630 | 
+| >100k-900k | $0.0025650 | $0.0032060 | $0.0025650 | $0.0032060 | 
+| >1.0M-4.5M | $0.0023090 | $0.0024530 | $0.0019620 | $0.0024530 | 
+| >4.5M-20M | $0.001962 | $0.002551 |
```
Comment (lizokm) [inline docs/pricing/index.mdx:81]:
```suggestion
| >4.5M | $0.001962 | $0.002551 |
```

Comment (lizokm) [inline docs/pricing/index.mdx:81]:
```suggestion
| Total Replays Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid
| Business Pay-as-you-go |
| >50-5k | $0.0030000 | $0.0037500 | $0.0030000 | $0.0037500 | 
| >5k-100k | $0.0028500 | $0.0035630 | $0.0028500 | $0.0035630 | 
| >100k-900k | $0.0025650 | $0.0032060 | $0.0025650 | $0.0032060 | 
| >900k-4.5M | $0.0023090 | $0.0028860 | $0.0023090 | $0.0028860 | 
| >4.5M | $0.0019620 | $0.0024530 | $0.0019620 | $0.0024530 |
```

File: docs/pricing/index.mdx [hunk 5]
```diff
@@ -72,21 +70,26 @@ Performance Units can be used towards multiple Performance event types, dependin
...
-| >100k-900k | $0.002565 | $0.003335 |
-| >900k-4.5M | $0.002309 | $0.003001 |
-| >4.5M-10M | $0.001962 | $0.002551 |
+| Total Replays Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid
+| Business Pay-as-you-go |
+| >5k-10k | $0.0028500 | $0.0035630 | $0.0028500 | $0.0035630 | 
+| >10k-100k | $0.0028500 | $0.0035630 | $0.0028500 | $0.0035630 | 
+| >100k-900k | $0.0025650 | $0.0032060 | $0.0025650 | $0.0032060 | 
+| >1.0M-4.5M | $0.0023090 | $0.0024530 | $0.0019620 | $0.0024530 | 
+| >4.5M | $0.001962 | $0.002551 |
```
Comment (lizokm) [inline docs/pricing/index.mdx:79]:
```suggestion
| Total Replays Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid
| Business Pay-as-you-go |
| >50-5k | $0.0030000 | $0.0037500 | $0.0030000 | $0.0037500 | 
| >5k-100k | $0.0028500 | $0.0035630 | $0.0028500 | $0.0035630 | 
| >100k-900k | $0.0025650 | $0.0032060 | $0.0025650 | $0.0032060 | 
| >900k-4.5M | $0.0023090 | $0.0028860 | $0.0023090 | $0.0028860 | 
| >4.5M | $0.0019620 | $0.0024530 | $0.0019620 | $0.0024530 |
```

**Comment by lizokm** [comment]
@a-hariti Any idea why my build is failing??

File: docs/pricing/index.mdx [hunk 6]
```diff
@@ -57,37 +57,37 @@ Tracing is enabled by spans and we bill per span unit.
...
 Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a pay-as-you-go budget that can be shared across all event types.
 
 | Team Pay-as-you-go | Business Pay-as-you-go |
-| $0.7800000 | $0.7800000 |
+| ------------------ | ---------------------- |
+| $0.7800000         | $0.7800000             |
 
 #### Attachments Pricing
 
 | Total Attachments Volume | Team Prepaid | Team Pay-as-you-go | Business Prepaid | Business Pay-as-you-go |
```
Comment (a-hariti) [inline docs/pricing/index.mdx:87]: What does the second row on this table mean?

File: docs/pricing/index.mdx [hunk 7]
```diff
@@ -1,180 +1,165 @@
...
 
-If you decide to increase your on-demand budget mid-month, Sentry will start processing additional data as soon as the increase becomes effective.
+\*In most cases, new budgets will take effect within minutes. Otherwise, we guarantee a maximum turnaround time of 24 hours.
 
-If you want to decrease your on-demand budget mid-month, but have already consumed more data than the new desired amount allows, your budget will be lowered to match what you’ve already used and any new data will be rejected. Your on-demand budget will be decreased to the desired amount on your next billing date.
+## Plans and Free Trials
 
-If you upgrade from a Team to a Business plan mid-cycle, any existing on-demand budget will be recalculated based on Business tier pricing. This may mean that your on-demand budget will be consumed faster than it was before.
+Sentry has one free Developer plan and three paid plans: Team, Business, and Enterprise. You can find more information about each of these plans and what they include on our [pricing page](https://sentry.io/pricing/).
```
Comment (lizokm) [inline docs/pricing/index.mdx:135]:
```suggestion
Sentry has one free Developer plan and three paid plans: Team, Business, and Enterprise. You can find more information about each by going to our [pricing page](https://sentry.io/pricing/).
```

File: docs/pricing/index.mdx [hunk 8]
```diff
@@ -1,180 +1,165 @@
...
-<Note>
+Anyone who's on a Team or Business plan can initiate a product trial directly from the Sentry UI. Keep in mind that each org can only trial each product once.
+During your 14-day product trial, you'll be able to send us an unlimited number of events. If you continue to use the product after your trial period is up, any additional usage will be billed using your available reserved volume or your pay-as-you-go budget. You won't be billed for any data you've used during the 14-day trial period.
 
-You must have either `Billing` or `Owner` permissions to access or make changes to information on the **Subscription** page. If you don't have those permissions, or you are not a member of the organization in Sentry, please reach out to a Billing member or an Owner to obtain invoices.
+### Who's Eligible for a Product Trial and When?
 
-</Note>
+**New accounts**
+New accounts will be able to request a one-time product trial on day 15, after the initial 14-day Business trial has ended, or immediately after a plan has been upgraded to a paid plan.
```
Comment (lizokm) [inline docs/pricing/index.mdx:153]:
```suggestion
**New accounts**

New accounts will be able to request a one-time product trial on day 15, after the initial 14-day Business trial has ended, or immediately after a plan has been upgraded to a paid plan.
```

File: docs/pricing/index.mdx [hunk 9]
```diff
@@ -1,180 +1,165 @@
...
 
-## Plans & Free Trials
+**Free accounts**
+Free accounts will be able to request a one-time product trial at any time.
```
Comment (lizokm) [inline docs/pricing/index.mdx:156]:
```suggestion
**Free accounts**

Free accounts will be able to request a one-time product trial at any time.
```

File: docs/pricing/index.mdx [hunk 10]
```diff
@@ -1,180 +1,166 @@
 ---
 title: Pricing & Billing
 sidebar_order: 1
-description: "Learn about pricing variables, managing volume, and different Sentry plans."
+description: "Learn about pricing, managing volume, and different Sentry plans."
```
Comment (lizokm) [inline docs/pricing/index.mdx:4]:
```suggestion
description: "Learn about pricing, managing volume, and the different Sentry plans."
```

File: docs/pricing/index.mdx [hunk 11]
```diff
@@ -1,180 +1,161 @@
...
 ---
 
 This page gives you an in-depth understanding of our pricing and billing structure, including:
 
-- [How our pricing works](#pricing-how-it-works) and [per-category-pricing](#per-category-pricing)
-- [Reserved (prepaid)](#prepaid-reserved-capacity) and [on-demand](#on-demand-capacity) volume
-- [Billing periods](#billing-cycles)
+- [How our pricing works](#pricing-how-it-works) and [per-category-pricing](#pricing-by-product-and-data-category)
+- [Reserved](/pricing/#prepaid-reserved-capacity) and [pay-as-you-go](/pricing/#pay-as-you-go) budget
+- [Billing cycles](#billing-cycles)
```
Comment (lizokm) [inline docs/pricing/index.mdx:11]:
```suggestion
- [How our pricing works](#pricing-how-it-works) and [per-category-pricing](#pricing-by-product-and-data-category)
- [Billing cycles](#billing-cycles)
```

File: docs/pricing/index.mdx [hunk 12]
```diff
@@ -1,180 +1,163 @@
 ---
 title: Pricing & Billing
 sidebar_order: 1
-description: "Learn about pricing variables, managing volume, and different Sentry plans."
+description: "Learn about pricing, managing volume, and the different Sentry plans."
 ---
 
+<Alert>
+This pricing page only applies to plans **started on June 5th, 2024 or later**. For information about older plans, go to your [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and click on "Subscription" (under the "Usage & Billing" heading.)
```
Comment (vivianyentran) [inline docs/pricing/index.mdx:8]:
```suggestion
This pricing page only applies to plans **started on June 5th, 2024 or later**. For information about older plans, go to your [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and click on "Subscription" (under the "Usage & Billing" heading).
```

File: docs/pricing/index.mdx [hunk 13]
```diff
@@ -1,180 +1,163 @@
...
+
+- **Reserved volume:** A predetermined amount of data you pay for in advance at a discount, on a monthly or annual basis. Your base plan includes a set amount of volume for each data category.
 
-We bill based on the amount of data you want us to process. Our [Developer, Team, and Business plans](https://sentry.io/pricing/), each come with a pre-set number of events that are included in the price.
+- **Pay-as-you-go budget:** Setting a pay-as-you-go budget covers any overages and ensures that you don't lose monitoring after your prepaid volume has run out. This is the maximum you'll pay for overages each month.
 
-On top of that, you can choose to pay for a set, prepaid volume of additional data at a discounted price. You can also set aside an on-demand budget to cover potential overages, which will only be used once the prepaid budget has run out.
+## Pricing: How It Works
+
+We bill based on the amount of data you want us to process. Our [Pricing plans](https://sentry.io/pricing/), each come with a pre-set event volume which is included in the price. For example, all paid plans come with a monthly set of:
```
Comment (vivianyentran) [inline docs/pricing/index.mdx:27]:
```suggestion
We bill based on the amount of data you want us to process. Our [Pricing plans](https://sentry.io/pricing/) each come with a pre-set event volume, which is included in the price. For example, all paid plans come with a monthly set of:
```

File: docs/pricing/index.mdx [hunk 14]
```diff
@@ -1,180 +1,163 @@
...
-![Reserved Volume settings in Subscriptions](./img/reserved-volume.png)
+#### Attachments Pricing
 
-For more information on how to manage your monthly spending, check out our [Quota Management documentation](/pricing/quotas/).
+| Attachments Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
+| ------------------ | ------------ | ---------- | ---------------- | -------------- |
+| >1G                | $0.2500000   | $0.3125000 | $0.2500000       | $0.3125000     |
 
-## Volume
+_** PAYG - pay-as-you-go.**_
```
Comment (vivianyentran) [inline docs/pricing/index.mdx:92]:
```suggestion
_* PAYG stands for pay-as-you-go._
```
I think this should be formatted as a footnote to explain what the * in the tables mean

File: docs/pricing/index.mdx [hunk 15]
```diff
@@ -1,180 +1,163 @@
...
+\*In most cases, new budgets will take effect within minutes. Otherwise, we guarantee a maximum turnaround time of 24 hours.
 
-### Changing Plans Mid-cycle
+## Plans and Free Trials
 
-If you decide to increase your on-demand budget mid-month, Sentry will start processing additional data as soon as the increase becomes effective.
+Sentry has one free Developer plan and three paid plans: Team, Business, and Enterprise. You can find more information about each by going to our [pricing page](https://sentry.io/pricing/).
 
-If you want to decrease your on-demand budget mid-month, but have already consumed more data than the new desired amount allows, your budget will be lowered to match what you’ve already used and any new data will be rejected. Your on-demand budget will be decreased to the desired amount on your next billing date.
+Plan upgrades take effect immediately. Plan downgrades and cancellations are processed as the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current contract.
```
Comment (vivianyentran) [inline docs/pricing/index.mdx:132]:
```suggestion
Plan upgrades take effect immediately. Plan downgrades and cancellations are processed at the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current contract.
```

**Review by vivianyentran — APPROVED** [review]
Added a couple edits but overall LGTM!

File: docs/pricing/index.mdx [hunk 16]
```diff
@@ -1,180 +1,163 @@
...
 
-If you run out of **both** your prepaid and on-demand volume, any new data you send will be rejected and you won't be charged for it.
+Any data sent after you've run through your reserved volume and pay-as-you-go budget will be dropped and you won't be charged for it. This also means that you'll lose monitoring for the remainder of the billing cycle, so make sure your budget matches your usage.
 
-### Per-Category Pricing
+### Pricing by Product and Data Category
 
-If you'd like to buy either prepaid or on-demand data on top of what's included in your plan, you can buy as much as you think you'll need per data category. [Errors](#errors-pricing), [Performance](#performance-pricing), [Replays](#replays-pricing), [Cron Monitors](#cron-monitors-pricing), and [Attachments](#attachments-pricing) each have their own unique pricing structures, as listed in the tables below:
+Sentry offers several products, each of which have their own pricing structure as listed in the tables below. Paid pricing plans include a standard volume of events for each product or data category and you can buy more either by prepaying for reserved volume and/or setting up a pay-as-you-go budget.
```
Comment (lizokm) [inline docs/pricing/index.mdx:40]:
```suggestion
Sentry offers several products, each of which have their own pricing structure as listed in the tables below. Paid pricing plans include a standard volume of events for each product or data category and you can buy more either by prepaying for reserved volume and/or setting up a pay-as-you-go (PAYG) budget.
```

File: docs/pricing/index.mdx [hunk 17]
```diff
@@ -1,180 +1,163 @@
 ---
 title: Pricing & Billing
 sidebar_order: 1
-description: "Learn about pricing variables, managing volume, and different Sentry plans."
+description: "Learn about pricing, managing volume, and the different Sentry plans."
 ---
 
+<Alert>
+This pricing page only applies to plans **started on June 5th, 2024 or later**. For information about older plans, go to your [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and click on "Subscription" (under the "Usage & Billing" heading).
+</Alert>
```
Comment (lizokm) [inline docs/pricing/index.mdx:9]:
```suggestion
This pricing only applies to free plans or paid plans **started on June 11th, 2024 or later**. For information about older plans, go to your [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and click on "Subscription" (under the "Usage & Billing" heading).
</Alert>
```

File: docs/pricing/index.mdx [hunk 18]
```diff
@@ -1,180 +1,163 @@
...
 
-## Pricing: How It Works
+## Terminology
+
+- **Volume:** Volume refers to the total number of events and attachments — or the total amount of data sent by your organization.
+
+- **Reserved volume:** A predetermined amount of data you pay for in advance at a discount, on a monthly or annual basis. Your base plan includes a set amount of volume for each data category.
 
-We bill based on the amount of data you want us to process. Our [Developer, Team, and Business plans](https://sentry.io/pricing/), each come with a pre-set number of events that are included in the price.
+- **Pay-as-you-go budget:** Setting a pay-as-you-go budget covers any overages and ensures that you don't lose monitoring after your prepaid volume has run out. This is the maximum you'll pay for overages each month.
```
Comment (lizokm) [inline docs/pricing/index.mdx:23]:
```suggestion
- **Pay-as-you-go budget:** Setting a pay-as-you-go (PAYG) budget covers any overages and ensures that you don't lose monitoring after your prepaid volume has run out. This is the maximum you'll pay for overages each month.
```

File: docs/pricing/index.mdx [hunk 19]
```diff
@@ -1,180 +1,163 @@
...
 
-## Pricing: How It Works
+## Terminology
+
+- **Volume:** Volume refers to the total number of events and attachments — or the total amount of data sent by your organization.
+
+- **Reserved volume:** A predetermined amount of data you pay for in advance at a discount, on a monthly or annual basis. Your base plan includes a set amount of volume for each data category.
 
-We bill based on the amount of data you want us to process. Our [Developer, Team, and Business plans](https://sentry.io/pricing/), each come with a pre-set number of events that are included in the price.
+- **Pay-as-you-go budget:** Setting a pay-as-you-go (PAYG) budget covers any overages and ensures that you don't lose monitoring after your prepaid volume has run out. This is the maximum you'll pay for overages each month.
```
Comment (lizokm) [inline docs/pricing/index.mdx:23]:
```suggestion
- **Pay-as-you-go budget:** Setting a pay-as-you-go (PAYG) budget covers any overages and ensures that you don't lose monitoring after your reserved volume has run out. This is the maximum you'll pay for overages each month.
```

File: docs/pricing/index.mdx [hunk 20]
```diff
@@ -1,180 +1,163 @@
...
 
 Please note that all per-category prices listed below have been rounded to the nearest millionth.
 
 </Note>
 
-{/*<!-- prettier-ignore-start -->*/}
-
 #### Errors Pricing
 
+| Errors Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
```
Comment (lizokm) [inline docs/pricing/index.mdx:50]:
```suggestion
| Errors Volume | Team Reserved | Team PAYG* | Business Reserved | Business PAYG* |
```

File: docs/pricing/index.mdx [hunk 21]
```diff
@@ -1,180 +1,163 @@
...
 
-| Total Replays Volume | Team/Business Prepaid | Team/Business On-Demand |
-|---|---|---|
-| >500-5k | n/a: minimum 10k volume required | $0.003900 |
-| >5k-10k | n/a: minimum 10k volume required | $0.003705 |
-| >10k-100k | $0.002850 | $0.003705 |
-| >100k-900k | $0.002565 | $0.003335 |
-| >900k-4.5M | $0.002309 | $0.003001 |
-| >4.5M-10M | $0.001962 | $0.002551 |
+| Span Unit Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
```
Comment (lizokm) [inline docs/pricing/index.mdx:62]:
```suggestion
| Span Unit Volume | Team Reserved | Team PAYG | Business Reserved | Business PAYG |
```

File: docs/pricing/index.mdx [hunk 22]
```diff
@@ -1,180 +1,163 @@
...
-
-| Total Attachments Volume | Team/Business Prepaid | Team/Business On-Demand |
-|---|---|---|
-| >25 - 100 (GB) | $0.24 | $0.312 |
-| >100 - 200 (GB) | $0.24 | $0.312 |
-| >200 - 10k (GB) | $0.23 | $0.299 |
+#### Replays Pricing
 
-{/*<!-- prettier-ignore-end -->*/}
+| Replays Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
```
Comment (lizokm) [inline docs/pricing/index.mdx:70]:
```suggestion
| Replays Volume | Team Reserved | Team PAYG | Business Reserved | Business PAYG |
```

File: docs/pricing/index.mdx [hunk 23]
```diff
@@ -1,180 +1,163 @@
...
-Use the slider on the [pricing](https://sentry.io/pricing/) page to determine the amount of prepaid data you might need based on the volume of event types you send.
+| Team PAYG* | Business PAYG* |
+| ---------- | -------------- |
+| $0.7800000 | $0.7800000     |
 
-![Reserved Volume settings in Subscriptions](./img/reserved-volume.png)
+#### Attachments Pricing
 
-For more information on how to manage your monthly spending, check out our [Quota Management documentation](/pricing/quotas/).
+| Attachments Volume | Team Prepaid | Team PAYG* | Business Prepaid | Business PAYG* |
```
Comment (lizokm) [inline docs/pricing/index.mdx:88]:
```suggestion
| Attachments Volume | Team Reserved | Team PAYG | Business Reserved | Business PAYG |
```

File: docs/pricing/index.mdx [hunk 24]
```diff
@@ -1,180 +1,163 @@
...
+
+- **Reserved volume:** A predetermined amount of data you pay for in advance at a discount, on a monthly or annual basis. Your base plan includes a set amount of volume for each data category.
 
-We bill based on the amount of data you want us to process. Our [Developer, Team, and Business plans](https://sentry.io/pricing/), each come with a pre-set number of events that are included in the price.
+- **Pay-as-you-go budget:** Setting a pay-as-you-go (PAYG) budget covers any overages and ensures that you don't lose monitoring after your reserved volume has run out. This is the maximum you'll pay for overages each month.
 
-On top of that, you can choose to pay for a set, prepaid volume of additional data at a discounted price. You can also set aside an on-demand budget to cover potential overages, which will only be used once the prepaid budget has run out.
+## Pricing: How It Works
+
+We bill based on the amount of data you want us to process. Our [Pricing plans](https://sentry.io/pricing/) each come with a pre-set event volume, which is included in the price. For example, all paid plans come with a monthly set of:
```
Comment (lizokm) [inline docs/pricing/index.mdx:27]:
```suggestion
We bill based on the amount of data processed. Each [paid plan](https://sentry.io/pricing/) comes with the below pre-set monthly event volume, which is included in the price:
```

File: docs/pricing/index.mdx [hunk 25]
```diff
@@ -1,180 +1,163 @@
...
+We bill based on the amount of data processed. Each [paid plan](https://sentry.io/pricing/) comes with the below pre-set monthly event volume, which is included in the price:
+- 50k errors
+- 10M span units
+- 50 replays
+- 1 cron monitor
+- 1GB of attachments
 
-- **[Prepaid volume:](#prepaid-reserved-capacity)** A predetermined amount of data you pay for in advance at a discount, on a monthly or annual basis.
-- **[On-demand volume:](#on-demand-capacity)** An overage budget that kicks in only if your prepaid volume has been used up. Having this makes it possible to control the maximum amount you will spend for overages each month.
+On top of that, you can choose to pay for a set, reserved volume of additional data at a discounted price. Any unused reserved volume will expire at the end of each billing month. You can also set aside a pay-as-you-go budget that can be used to cover potential overages in any data category. Your pay-as-you-go budget will only be accessed once the reserved budget has run out and you'll only be billed for what you use (up to the maximum you've set).
```
Comment (lizokm) [inline docs/pricing/index.mdx:34]:
```suggestion
On top of that, you can choose to pay for a set, reserved volume of additional data at a discounted price. Any unused reserved volume will expire at the end of each billing month. You can also set aside a pay-as-you-go budget that can be used to cover potential overages in any data category. Your pay-as-you-go budget will only be accessed once the reserved volume has run out and you'll only be billed for what you use (up to the maximum you've set).
```

File: docs/pricing/index.mdx [hunk 26]
```diff
@@ -1,180 +1,163 @@
...
 
-If you want to decrease your on-demand budget mid-month, but have already consumed more data than the new desired amount allows, your budget will be lowered to match what you’ve already used and any new data will be rejected. Your on-demand budget will be decreased to the desired amount on your next billing date.
+Plan upgrades take effect immediately. Plan downgrades and cancellations are processed at the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current contract.
 
-If you upgrade from a Team to a Business plan mid-cycle, any existing on-demand budget will be recalculated based on Business tier pricing. This may mean that your on-demand budget will be consumed faster than it was before.
+All new Sentry accounts come with a 14-day free trial period. You'll have the option to upgrade and continue to use Sentry's latest features like Insights at any time before your trial ends. Upgrades will be effective immediately. Users who haven't upgraded by the end of their free trial will remain on the free Developer plan with limited features.
```
Comment (lizokm) [inline docs/pricing/index.mdx:134]:
```suggestion
All new Sentry accounts come with a 14-day free trial period. You'll have the option to upgrade and continue to use Sentry's paid features at any time before your trial ends. Upgrades will be effective immediately. Users who haven't upgraded by the end of their free trial will remain on the free Developer plan with limited features.
```

File: docs/pricing/index.mdx [hunk 27]
```diff
@@ -1,180 +1,163 @@
...
+All new Sentry accounts come with a 14-day free trial period. You'll have the option to upgrade and continue to use Sentry's paid features at any time before your trial ends. Upgrades will be effective immediately. Users who haven't upgraded by the end of their free trial will remain on the free Developer plan with limited features.
 
-In most cases, new budgets will take effect within minutes. Otherwise, we guarantee a maximum turnaround time of 24 hours.
+## Product Trials
 
-## Billing Periods
+In addition to the free Business trial available to new accounts, Sentry offers a one-time, 14-day product trial as a way to let your organization try out products that aren't part of your billing plan (such as Tracing and Session Replays). Product trials are specific to each product, so you can try one product at a time or multiple products at the same time.
 
-Two different billing periods are available:
+Anyone who's on a Team or Business plan can initiate a product trial directly from the Sentry UI. Keep in mind that each org can only trial each product once.
```
Comment (lizokm) [inline docs/pricing/index.mdx:140]:
```suggestion
Anyone who's on a Free, Team, or Business plan can initiate a product trial directly from the Sentry UI. Keep in mind that each org can only trial each product once.
```

File: docs/pricing/index.mdx [hunk 28]
```diff
@@ -1,180 +1,163 @@
...
 
 <Note>
+  You have to have Billing or Owner-level permissions to make changes to
+  information on the Subscription page or access invoices. If you need an
+  invoice, but don't have the right permissions, reach out to your
+  administrator.
+</Note>
 
-If you've enabled [Spend Allocation](/pricing/quotas/spend-allocation/), and your project has consumed both its dedicated event allocation and your organization's un-allocated pool, any excess event consumption will come from your on-demand budget **even if you have reserved volume still available**.
+We run on two billing cycles - one for paid plans, and the other for pay-as-you-go budget. If you're on a paid plan, you can choose to be billed for it and any reserved volume on either a monthly or an annual basis.
```
Comment (lizokm) [inline docs/pricing/index.mdx:112]:
```suggestion
Sentry uses two billing cycles - one for paid plan subscriptions and reserved volume, and the other for pay-as-you-go usage. If you're on a paid plan, you can choose to be billed for it on either a monthly or an annual basis.
```

File: docs/pricing/index.mdx [hunk 29]
```diff
@@ -1,180 +1,163 @@
...
+  information on the Subscription page or access invoices. If you need an
+  invoice, but don't have the right permissions, reach out to your
+  administrator.
+</Note>
 
-If you've enabled [Spend Allocation](/pricing/quotas/spend-allocation/), and your project has consumed both its dedicated event allocation and your organization's un-allocated pool, any excess event consumption will come from your on-demand budget **even if you have reserved volume still available**.
+Sentry uses two billing cycles - one for paid plan subscriptions and reserved volume, and the other for pay-as-you-go usage. If you're on a paid plan, you can choose to be billed for it on either a monthly or an annual basis.
 
-</Note>
+If you've added a pay-as-you-go budget and you're on a monthly billing plan, you'll be billed for any pay-as-you-go usage within that same billing cycle. If you're on an annual billing plan, any pay-as-you-go usage will be billed separately on a monthly basis.
```
Comment (lizokm) [inline docs/pricing/index.mdx:114]:
```suggestion
- **Subscription billing cycle:** Billing cycle for your paid plan and reserved volume for which you can elect to be billed on either a monthly or an annual basis.
- **Pay-as-you-go billing cycle:** Billing cycle for your pay-as-you-go usage, for which you'll be billed on a monthly basis. 

_If you're on a monthly subscription billing cycle, your pay-as-you-go billing cycle will be aligned to it._
```

**Review by lizokm — COMMENTED** [review]
Comment

**Pull request merged**